portcast/api/openapi.yaml

```diff
@@ -2836,6 +2845,14 @@ components:
         vessel_imo:
           format: nullable
           type: integer
+        mode_of_transport:
+          enum:
+          - TRUCK
+          - RAIL
+          - VESSEL
+          - BARGE
+          - UNCLASSIFIED
+          type: string
```

Comment:
is this for sure or did you just take the one we currently support?

Reply:
Is come form the portcast documentation https://portcast.stoplight.io/docs/portcast-api/branches/main/d3abcc88fa3a8-container-event